governance/polls/SparkLend Gnosis - Update sxDAI onboard parameters  - April 15, 2024.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SparkLend Gnosis - Update sxDAI onboard parameter - April 15, 2024
+summary: Update the "Oracle" parameter for onboarding sxDAI (Savings xDAI).
+discussion_link: https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-04-15T16:00:00
+end_date: 2024-04-18T16:00:00
+---
+# Poll: SparkLend Gnosis - Update sxDAI onboard parameters - April 15, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Tuesday, April 15 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to updating the "Oracle" parameter for sxDAI (Savings xDAI) onto SparkLend on Gnosis. The onboarding will occur with the parameters below:
+
+### Spark Parameters
+
+* Token: sxDAI
+* Token Address: [0xaf204776c7245bF4147c2612BF6e5972Ee483701](https://gnosisscan.io/address/0xaf204776c7245bF4147c2612BF6e5972Ee483701)
+* Oracle: Set to a fixed rate of 1 USD
+* Oracle Provider: Chainlink
```

Comment:
Chainlink won't be the provider if we are using a fixed oracle
```suggestion
* Oracle Provider: N/A
```

Reply:
Non-blocking but You could remove the rest of the parameters and just highlight the one that is changing to make it clearer.

Reply:
Since the token has not yet been onboarded, I think it would be beneficial to reiterate the previously approved parameters, including those approved by the Stability Facilitator. This way, voters can see the final parameters before making a decision to change the oracle